.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "pub"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    versioning-strategy: "widen"
+    # Limit open pull requests to 1 to avoid multiple PRs for the same dependency.
+    open-pull-requests-limit: 1
```

Comment:
Suggestion: Set this to 5, it is OK to have multiple requests open here.

Update the comment as well.